.ci_support/osx_64_.yaml

```diff
@@ -31,7 +31,7 @@ fftw:
 fortran_compiler:
 - gfortran
 fortran_compiler_version:
-- '9'
+- '11'
```

Comment:
This change from the rerenderer is breaking osx-64.  I don't understand it, but we should figure this out for 5.0.0.